src/github.py

```diff
@@ -0,0 +1,62 @@
+import os
+import requests
+import streamlit as st
+
+ORG = 'NYCPlanning'
```

Comment:
I don't think you have black formatting activated as we are mixing single quotes with double quotes based on other scripts in this repo, I think if we update the devcontainer.json file with the proper extensions based on the [template repo](https://github.com/NYCPlanning/db-template-repo/blob/main/.devcontainer/devcontainer.json)

Reply:
Gotcha. I'll do that. @damonmcc good example for my to try trunk?

Reply:
happy to troubleshoot getting black to do the formatting! (only on files changed in this PR lol)

Reply:
Just used black for now